tests/test_runner.sh

```diff
@@ -5,7 +5,7 @@ cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
 
 # Default to using /var/tmp for test space, since it's more likely to support
 # labels than /tmp, which is often on tmpfs.
-export TMPDIR=${TMPDIR:-/var/tmp}
+export TMPDIR=${TMPDIR:-/tmp}
```

Comment:
The comment above says default /var/tmp so this no longer matches, I am not sure what the comment means by supporting labels, selinux? That should work on tmpfs so I don't think that is blocker